### ISSUE TITLE
Direct Platform Specs Group Removal

### ISSUE THREAD

**Issue by kcaramat** [Fiserv/Support#708]
### Product

Direct Platform Spec Tenant

### Page

Direct Platform Specification

### Description

Please remove the Groups. They are no longer needed.
MULTILINK_SPECIFICATIONS
NETWORK_TOKEN
OFFSITE_SPECS
TELECHECK
TRANSARMOR

### Expected Behaviour

Please remove the Groups. They are no longer needed.
MULTILINK_SPECIFICATIONS
NETWORK_TOKEN
OFFSITE_SPECS
TELECHECK
TRANSARMOR

### Operating system

N/A

### Browser

Chrome

**Comment by yanyan-cathy** [comment]
Hi, thank you for contacting us. A ticket is created for the DB change, and I will keep you updated once it is done.

**Comment by yanyan-cathy** [comment]
the implementation is done in https://firstdatateam.atlassian.net/browse/PORT-11356, close the issue now
